--- a/src/iface/interface/ipv6.rs	2026-06-24 23:16:09.010170369 +0800
+++ b/src/iface/interface/ipv6.rs	2026-06-17 10:32:35.684840656 +0800
@@ -170,6 +170,7 @@
     }
 
     /// Get the first link-local IPv6 address of the interface, if present.
+    #[allow(dead_code)]
     fn link_local_ipv6_address(&self) -> Option<Ipv6Address> {
         self.ip_addrs.iter().find_map(|addr| match *addr {
             #[cfg(feature = "proto-ipv4")]
@@ -562,6 +563,7 @@
         ))
     }
 
+    #[allow(dead_code)]
     pub(super) fn mldv2_report_packet<'any>(
         &self,
         records: &'any [MldAddressRecordRepr<'any>],
